articles/Q-language setup.md

```diff
@@ -0,0 +1,70 @@
+# Setup and workflow for Q# language topics from qsharp-language repo
```

Comment:
Should this be marked as excluded from the generated docs, so that we can keep it in the repo without affecting the actual content at docs.microsoft.com? Alternatively, this could maybe go on the Wiki for this repo?

Reply:
Yes, I should move it over to the wiki - thanks.